src/Datadog.Trace/SpanEventArgs.cs

```diff
@@ -2,12 +2,17 @@
 
 namespace Datadog.Trace
 {
-    internal class SpanEventArgs : EventArgs
+    /// <summary>
+    /// EventArgs for a Span
+    /// </summary>
+    public class SpanEventArgs : EventArgs
```

Comment:
Any particular reason we made this `public`?

Reply:
`AspScopeManager`  is now used in `AspNetIntegration` which is in another assembly, at least for now. If I get it all working and can move stuff back into the ...ClrProfiler.Managed assembly I will, and that's when I'd change back to Internal.

Reply:
Let's change this back to `internal` and give access to assemblies that need it.